src/ucp/proto/proto_am.c

```diff
@@ -23,12 +23,13 @@ static size_t ucp_proto_pack(void *dest, void *arg)
     return 0;
 }
 
-ucs_status_t ucp_proto_progress_am_bcopy_single(uct_pending_req_t *self)
+ucs_status_t ucp_proto_progress_am_bcopy_single(uct_pending_req_t *self,
+                                                ucs_status_t status)
```

Comment:
Further discussion with yossi led to the decision to remove the status argument from pending callbacks. I'll make the change and resubmit.